packages/test/test-end-to-end-tests/src/test/gc/gcLoadingFromOlderSummaries.spec.ts

```diff
@@ -39,6 +37,12 @@ describeNoCompat("GC loading from older summaries", (getTestObjectProvider) => {
 	let containerRuntime: IContainerRuntime;
 	let dataStoreA: ITestDataObject;
 
+	const settings = { "Fluid.ContainerRuntime.Test.CloseSummarizerDelayOverrideMs": 10 };
```

Comment:
It seems like the tests should await for at least time to ensure that the container closes. Otherwise, the test may become flaky.

Reply:
Change reverted